src/MoBi.Presentation/Mappers/ContainerToContainerDTOMapper.cs

```diff
@@ -11,16 +11,35 @@ public interface IContainerToContainerDTOMapper : IMapper<IContainer, ContainerD
 
    public class ContainerToContainerDTOMapper : ObjectBaseToObjectBaseDTOMapperBase, IContainerToContainerDTOMapper
    {
+      private readonly IObjectPathFactory _objectPathFactory;
       private readonly IIconRepository _iconRepository;
 
-      public ContainerToContainerDTOMapper(IIconRepository iconRepository)
+      public ContainerToContainerDTOMapper(IObjectPathFactory objectPathFactory, IIconRepository iconRepository)
       {
+         _objectPathFactory = objectPathFactory;
          _iconRepository = iconRepository;
       }
 
       public virtual ContainerDTO MapFrom(IContainer container)
       {
-         return MapContainer(container, new ContainerDTO());
+         var dto = MapContainer(container, new ContainerDTO());
+         UpdateParentPath(container, dto);
+         return dto;
+      }
+
+      protected void UpdateParentPath(IContainer container, ContainerDTO containerDTO)
```

Comment:
This is the meat of the change here. If there is a parent container, I set the ParentPath from the parent otherwise I allow edit and set it to the ParentPath